.circleci/config.yml

```diff
@@ -58,7 +55,9 @@ jobs:
   build-docs:
     <<: *linux
     steps:
-      - *setup_docker
+      - add_ssh_keys:
+          fingerprints:
+            - f8:26:86:86:f8:d3:a5:66:ea:7d:f6:42:2e:5c:7a:82
```

Comment:
@daler actually using custom SSH keys seems pretty easy in circleci compared to what we had before. Very convenient.

Reply:
Oh, that is really convenient!